platform.md

```diff
@@ -840,6 +840,9 @@ Usage:
       - `run-image.reference` SHALL uniquely identify `<run-image>`
       - `run-image.top-layer` SHALL be set to the uncompressed digest of the top layer in `<run-image>`
     - The value of `io.buildpacks.stack.*` labels SHALL be modified to that of the new `run-image`
+- **If** the provided `<run-image>` is not found in `stack.run-image.image` or `stack.run-image.mirrors`:
```

Comment:
Sorry this might be a dumb question, but does this mean every new run image released needs to have some sort of mapping/record/reference indicated in the stack?

Reply:
I think it depends - if you want `stack.run-image.mirrors` to have data, then yes the mapping must exist. Worth noting: future versions of the spec do away with stack.toml in favor of run.toml, which now contains a list of run images (and associated mirrors for each). More information could be found here: https://github.com/buildpacks/spec/pull/335/files#diff-e603760990971da3f77be4bb8d77c3405098f006814fd8c054d2d15f395b8330R934-R939 and here: https://github.com/buildpacks/spec/pull/335/files#diff-e603760990971da3f77be4bb8d77c3405098f006814fd8c054d2d15f395b8330R1238

@hernandanielg I'm realizing that this PR should probably be pointed at https://github.com/buildpacks/spec/pull/335 due to the aforementioned changes. Are you able to rebase? There will likely be a merge conflict.

Reply:
Thank you for the clarification!